### PR TITLE
HPCC-31188 Ensure scopes table display "a" value.

### PR DESCRIPTION
**Pull request by GordonSmith**
Temp fix to ensure either "value", "max" or "avg" is displayed for selected columns.

Add TimeLocalExecute to the default list of columns.

Gordon Smith <GordonJSmith@gmail.com>

## Type of change:
- [x] This change is a bug fix (non-breaking change which fixes an issue).
- [ ] This change is a new feature (non-breaking change which adds functionality).
- [ ] This change improves the code (refactor or other change that does not change the functionality)
- [ ] This change fixes warnings (the fix does not alter the functionality or the generated code)
- [ ] This change is a breaking change (fix or feature that will cause existing behavior to change).
- [ ] This change alters the query API (existing queries will have to be recompiled)

## Checklist:
- [x] My code follows the code style of this project.
  - [ ] My code does not create any new warnings from compiler, build system, or lint.
- [x] The commit message is properly formatted and free of typos.
  - [ ] The commit message title makes sense in a changelog, by itself.
  - [x] The commit is signed.
- [ ] My change requires a change to the documentation.
  - [ ] I have updated the documentation accordingly, or...
  - [ ] I have created a JIRA ticket to update the documentation.
  - [ ] Any new interfaces or exported functions are appropriately commented.
- [x] I have read the CONTRIBUTORS document.
- [ ] The change has been fully tested:
  - [ ] I have added tests to cover my changes.
  - [ ] All new and existing tests passed.
  - [ ] I have checked that this change does not introduce memory leaks.
  - [ ] I have used Valgrind or similar tools to check for potential issues.
- [x] I have given due consideration to all of the following potential concerns:
  - [ ] Scalability
  - [ ] Performance
  - [ ] Security
  - [ ] Thread-safety
  - [ ] Cloud-compatibility
  - [ ] Premature optimization
  - [ ] Existing deployed queries will not be broken
  - [ ] This change fixes the problem, not just the symptom
  - [ ] The target branch of this pull request is appropriate for such a change.
- [ ] There are no similar instances of the same problem that should be addressed
  - [ ] I have addressed them here
  - [x] I have raised JIRA issues to address them separately
- [ ] This is a user interface / front-end modification
  - [ ] I have tested my changes in multiple modern browsers
  - [ ] The component(s) render as expected

## Smoketest:
- [ ] Send notifications about my Pull Request position in Smoketest queue.
- [ ] Test my draft Pull Request.

### CODE REVIEW

File: esp/src/src-react/components/Metrics.tsx [hunk 1]
```diff
@@ -181,7 +181,13 @@ export const Metrics: React.FunctionComponent<MetricsProps> = ({
                     (options.scopeTypes.indexOf(row.type) >= 0);
             }).map((row, idx) => {
                 row.__hpcc_id = row.name;
-                return [idx, row.type, row.name, ...options.properties.map(p => row[p] !== undefined ? row[p] : ""), row];
+                return [idx, row.type, row.name, ...options.properties.map(p => {
+                    return row[p] ||
+                        row.__groupedProps[p]?.Value ||
+                        row.__groupedProps[p]?.Max ||
+                        row.__groupedProps[p]?.Avg ||
+                        "";
```
Comment (GordonSmith) [inline esp/src/src-react/components/Metrics.tsx:189]: Change to `??` from `||`.  Will need:  https://github.com/hpcc-systems/Visualization/pull/4162

**Review by jeclrsg — APPROVED** [review]

File: esp/src/src-react/components/Metrics.tsx (hunk 1 above)
Comment (ghalliday) [reply esp/src/src-react/components/Metrics.tsx:189]: Can you clarify.  Is this something you plan to change in the future?  It looks like you have merged that changed now.
Should I merge this, or wait until you modify?

**Review by jeclrsg — APPROVED** [review]

**Comment by GordonSmith** [comment]
Closing and re-opening to force test re-run.

**Comment by GordonSmith** [comment]
Closing and re-opening to force re-test

**Pull request merged**